app/experimenter/visualization/api/v1/views.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from django.core.files.storage import default_storage
+from django.http import HttpResponse
```

Comment:
Oh Django Rest Framework has its own `Response` object:

[`Handler methods may return REST framework's Response, instead of Django's HttpResponse. The view will manage content negotiation and setting the correct renderer on the response.`](https://www.django-rest-framework.org/api-guide/views/#api_view)